sdk/servicebus/Azure.Messaging.ServiceBus/api/Azure.Messaging.ServiceBus.netstandard2.0.cs

```diff
@@ -425,6 +425,36 @@ public enum ServiceBusTransportType
         AmqpWebSockets = 1,
     }
 }
+namespace Azure.Messaging.ServiceBus.Amqp
+{
+    public enum AmqpBodyType
+    {
+        Unspecified = 0,
```

Comment:
With the change to make the AmqpDataBody populated even when using SBM.Body, do we still need Unspecified? I think it would only be set for no body, but I don't see the harm in just defaulting to Data in this case as the user won't get any data back anyway.

Reply:
As mentioned above, this isn't the situation. Body and Data do not have a direct relationship on ServiceBusMessage. To do that, it would require extra work which is unnecessary since the user can ask which body type the message was made from, and it will be accurate. Meaning, when it says Unspecified, they access it via Body. When it says Data, they access it via Data. We could change the naming here if that is an issue. I tried to keep the nomenclature related to AMQP and not ServiceBusMessage.

Reply:
I thought we were changing the behavior to allow GetAmqpDataBody to return the body even when message was created from the ctor as opposed to the extensions. Wasn't that the reason for capturing the DataCount?

Reply:
Unspecified has been removed. Data access for both sending/receiving messages works the same way now.